svd/extract.sh

```diff
@@ -3,8 +3,16 @@ for f in vendor/*.zip; do
     unzip -juLL $f '**.svd'
 done
 
-ln -s stm32h743x.svd stm32h743.svd || true
-ln -s stm32h743x.svd stm32h743v.svd || true
-ln -s stm32h753x.svd stm32h753.svd || true
-ln -s stm32h753x.svd stm32h753v.svd || true
-ln -s stm32h7x7_cm7.svd stm32h747cm7.svd || true
+rm stm32h743.svd || true
+rm stm32h743v.svd || true
+rm stm32h753.svd || true
+rm stm32h753v.svd || true
+rm stm32h747cm7.svd || true
+rm stm32h757cm7.svd || true
```

Comment:
Why remove these files only to recreate them immediately? The `cp` call should be fine overwriting them unless I'm missing something...

Reply:
Good point, these are vestigial from previous use of `ln -s`